requirements.txt

```diff
@@ -2,14 +2,18 @@
 # This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
-#    pip-compile requirements.in
+#    pip-compile
```

Comment:
Oh, you should probably run `pip-compile requirements.in` to use it properly

Reply:
Done, thank you.